setup.py

```diff
@@ -42,7 +42,7 @@
         ],
         # Note: This is only required for internal use
         "rtd": [
-            "myst_parser",
+            "myst_parser==0.11.2",
             "markdown-it-py~=0.4.5",
```

Comment:
this is why! you should remove this line as its a dependancy clash

Reply:
you can also unpin sphinx from <3 if you want 😄

Reply:
Have done both and works locally. Will open another pr and hopefully rtd will be happy. Thx a lot!

Reply:
No worries! I guess it might depend on what version of pip you have, as to how it resolves the conflict